tools/build_package.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+/*
+ * This file runs as part of the npm packaging process.
+ *
+ * It reads a set number of contracts from the truffle build directory and
+ * extracts ABI and BIN of each contract. The extracted information is added to
+ * a new object that is finally serialized to disk. That JSON file will be
+ * exported by this package.
+ *
+ * To add a contract to the published package, add its name to array of contract
+ * names.
+ */
+
+const fs = require('fs');
+
+const contractNames = [
```

Comment:
What do want the policy for exposed to be? All contracts to be deployed + interfaces?

If so then there are a few contracts here we can get rid of, as some of the libraries are completely inlined during compilation (and are essentially implementation details):
- `BytesLib`
- `CircularBufferUint`
- `GatewayBase`
- `RLP`
- `RLPEncode`
- `SafeMath`
- `SimpleStake`

Reply:
I am not sure. I removed them for now to keep it small. We can add them if for any reason someone needs them.
What about the interfaces?

Reply:
I think interfaces are usually very valuable for library authors, as they provide a clear destinction between what is the interface (which usually has higher stability guarantees) and what is a specific implementation of the interface.

Reply:
OK then I will leave them in. Thanks! 👍